src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    @Override
+    public String getPrintableString() {
```

Comment:
To avoid one extra append later for readability, return "address: "+ value would be better. Same for the rest.